packages/circuit-ui/components/NotificationToast/NotificationToast.tsx

```diff
@@ -0,0 +1,223 @@
+/**
+ * Copyright 2021, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  HTMLAttributes,
+  ReactNode,
+  RefObject,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import { css } from '@emotion/react';
+import { Alert, Confirm, Info, Notify } from '@sumup/icons';
+
+import styled, { StyleProps } from '../../styles/styled';
+import { useAnimation } from '../../hooks/useAnimation';
+import Body from '../Body';
+import { TrackingProps } from '../../hooks/useClickEvent';
+import CloseButton from '../CloseButton';
+import { ClickEvent } from '../../types/events';
+import { BaseToastProps, createUseToast } from '../ToastContext';
+
+const TRANSITION_DURATION = 200;
+const DEFAULT_HEIGHT = 'auto';
+
+type Variant = 'info' | 'confirm' | 'notify' | 'alert';
+
+export type NotificationToastProps = HTMLAttributes<HTMLDivElement> &
+  BaseToastProps & {
+    /**
+     * Notification toast variants. Defaults to `info`.
+     */
+    variant?: Variant;
+    /**
+     * Notification toast headline to provide information (optional)
+     */
+    headline?: string | ReactNode;
```

Comment:
I forgot that this could have a headline. Can we remove it?

- to assistive tech this will not be announced as a headline, anything in a live region is just raw text
- allowing for a headline encourages longer copy, we should do everything to keep toasts short (for sighted users since they auto-dismiss, and for SC users since they interrupt)

Reply:
Has this been discussed with the design team?

Reply:
It hasn't yet. We discussed with @amelako that it should be alright to keep the heading, since the semantics don't add a lot of context in this case (it's not describing a section).

@amelako could you update the other  threads with other things we discussed, for future reference? 🙂